libparsec/crates/platform_mountpoint/src/unix/inode.rs

```diff
@@ -0,0 +1,185 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+use std::{
+    collections::HashMap,
+    ffi::OsString,
+    path::{Path, PathBuf},
+    sync::{Mutex, RwLock},
+};
+
+use crate::MountpointManager;
+
```

Comment:
This inode stuff seems pretty complicated, we should have good unit tests on this !

Reply:
Done